src/cache/meta_data_cache.rs

```diff
@@ -81,7 +81,12 @@ impl TryFrom<&EsedbInfo<'_>> for MetaDataCache {
         let mut record_by_guid = HashMap::new();
         let mut root = None;
         //let mut root_dse = None;
-        let count = info.data_table().count_records()?;
+        let count = match info.data_table().count_records() {
+            Ok(x) => x,
+            Err(_) => {
+                info.data_table().count_records()? as i32
+            }
+        };
```

Comment:
@darkrael, I suggest:

```suggestion
        let count = libesedb_count(|| info.data_table().count_records())?;
```

while I implemented `libesedb_count` as a separate function like this:

```rust
pub(crate) fn libesedb_count(count_fn: impl Fn() -> io::Result<i32>) -> io::Result<i32> {
    match count_fn() {
        Ok(val) => Ok(val),
        Err(_) => count_fn()
    }
}
```

This way, we have the ugly mitigation at a single position in the code and can change this to a no-op as soon as the problem in libesedb gets fixed.